HACKING.md

```diff
@@ -1,10 +1,88 @@
-The `manifests/` directory contains OpenShift manifests that will set up a Jenkins pipeline
-using the [Kubernetes Jenkins plugin](https://github.com/jenkinsci/kubernetes-plugin).
+The `manifests/pipeline.yaml` file contains an OpenShift template that
+will set up a Jenkins pipeline using the
+[Kubernetes Jenkins plugin](https://github.com/jenkinsci/kubernetes-plugin).
 
-### Create a Jenkins instance with a persistent volume backing store:
+These instructions will cover both local developer workflows, as well as
+interacting with the production pipeline in CentOS CI. A section header
+may indicate that the section only applies to one of the two workflows.
 
+### [LOCAL] Set up an OpenShift cluster
+
+First, make sure to install `oci-kvm-hook` on your host system (not in a
+pet container). This is required to ensure that the pipeline has access
+to `/dev/kvm`:
+
+```
+rpm-ostree install oci-kvm-hook # if on Silverblue
```

Comment:
maybe we should say "ostree based system" instead of silverblue. I might run on Atomic Host today or FCOS tomorrow.